lib/common/to-string.ts

```diff
@@ -42,6 +42,8 @@ Zone.__load_patch('toString', (global: any, Zone: ZoneType) => {
     }
     return originalFunctionToString.apply(this, arguments);
   };
+  (newFunctionToString as any)[ORIGINAL_DELEGATE_SYMBOL] = originalFunctionToString;
```

Comment:
current version, we attached `originalFunctionToString` to `Zone`, and if we attached it to `Function.prototype`, we don't need to attached it to `Zone` any more.

so please just change this line https://github.com/gdh1995/zone.js/blob/b98f284030a4296a0e476728829298515edc3b0d/lib/common/to-string.ts#L14
```javascript
const originalFunctionToString = Function.prototype[ORIGINAL_DELEGATE_SYMBOL] =
      Function.prototype.toString;
```